src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetSupportsSharing/SharedMediaItem/DownloadUploadLink/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { Anchor, Text } from 'grommet';
+import React from 'react';
+import styled from 'styled-components';
+
+import { SharedLiveMedia, uploadState } from 'types/tracks';
+
+const StyledAnchor = styled(Anchor)`
+  cursor: ${({ isClickable }: { isClickable: boolean }) =>
+    isClickable ? 'pointer' : 'auto'};
+  font-family: Roboto-Medium;
+
+  &:hover {
+    text-decoration: ${({ isClickable }: { isClickable: boolean }) =>
+      isClickable ? 'underline' : undefined};
+  }
+`;
+
+interface DownloadUploadLinkProps {
+  sharedLiveMedia: SharedLiveMedia;
+  sharedLiveMediaTitle: string;
+}
+
+export const DownloadUploadLink = ({
```

Comment:
This component's purpose is to display the name of the uploaded file, and allow its downloading by clicking on this name. However, in case of error, this component is also responsible for displaying a message indicating the upload has failed, and in this case the message displayed shouldn't be clickable.    

To make this clearer, i renamed the component `TitleDisplayer` in commit [99eed90a](https://github.com/openfun/marsha/pull/1528/commits/99eed90aafd03703f175ebfd5fff9af8aadc0117) and [bf22bb3](https://github.com/openfun/marsha/pull/1528/commits/bf22bb30b5c58a3fa449cd3cf6c0b4d40e4a4eab)